README.md

```diff
@@ -83,25 +83,40 @@ _Video.js uses [BrowserStack][browserstack] for compatibility testing._
 
 Video.js is [licensed][license] under the Apache License, Version 2.0.
 
-
 [browserstack]: https://browserstack.com
-[buildwith]: https://trends.builtwith.com/media/VideoJS
+
```

Comment:
👎 to all this whitespace. If these links where visible as is, it might make more sense but it just makes maintaining these a bit worse, I think.